docs/_layouts/index.html

```diff
@@ -18,6 +18,9 @@ <h1 class="display-4 mb-0">{{ page.title }}</h1>
             ({{ formatted_date }})
             </em>
         </p>
+        <div class="mt-3">
+            <a href="/single-page.html" class="btn btn-primary btn-sm" target="_blank" rel="noopener">Show single page</a>
```

Comment:
rather than 'single page', can we call this 'Print-ready copy', perhaps adding an icon? e.g. `bi-printer`